persistent/persistent.cabal

```
@@ -1,5 +1,5 @@
 name:            persistent
-version:         2.14.1.0
+version:         2.14.1.1
```

Since we're exporting a new term, this is a minor version bump, according to PVP

```suggestion
version:         2.14.2.0
```